package.json

```diff
@@ -2,12 +2,13 @@
   "name": "@openmrs/esm-root-config",
   "version": "1.1.2",
   "description": "A javascript module for setting up and configuring your SPA ",
-  "main": "dist/openmrs-esm-root-config.lib.js",
+  "main": "dist/openmrs-esm-root-config.defaults.js",
```

Comment:
`openmrs-esm-root-config.lib.js` exports a number of helper functions to be used in distro root configs. When an organization needs to add custom apps to their distro, they can extend the `openmrs-esm-root-config` using some of the functions provided in that file. Removing it from `main` means they can't be imported and used. @joeldenning   is this right?

Reply:
That is right.

esm-root-config is a weird one for packmap -- it should actually point to the distro-specific root config that in turn has the `@openmrs/esm-root-config` package as a dependency.

Reply:
Got it. So should we move `openmrs-esm-root-config.defaults.js` into its own module?

Reply:
I don't think so

Reply:
I think we should have some way for an implementation to get SPA running on their server without writing their own root config override

Reply:
Oh yeah, I guess that's easy enough to solve just by editing the import map after generation. Closing.